slackbot/dispatcher.py

```diff
@@ -177,3 +185,7 @@ def channel(self):
     @property
     def body(self):
         return self._body
+
+    def docs_reply(self):
+        reply = ['    • `{0}` {1}'.format(v.__name__, v.__doc__ or "") for p, v in iteritems(self._plugins.commands['respond_to'])]
```

Comment:
+1 I hate that the default reply is documentation, which mucks up the channel :(